### PR TITLE
add ClusterResourceQuota template+revision in NSTemplateTier CR

### PR DESCRIPTION
**Pull request by xcoulon**
files generated after changes in https://github.com/codeready-toolchain/api/pull/116

Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: deploy/registration-service/registration-service.yaml [hunk 1]
```diff
@@ -31,13 +31,6 @@ objects:
       - masteruserrecords
       verbs:
       - get
-    - apiGroups:
-      - toolchain.dev.openshift.com
-      resources:
-      - bannedusers
-      verbs:
-      - get
-      - list
```
Comment (MatousJobanek) [inline deploy/registration-service/registration-service.yaml]: Please, revert these changes - it's because of https://github.com/codeready-toolchain/host-operator/pull/151#issuecomment-594006688

Comment (xcoulon) [reply deploy/registration-service/registration-service.yaml]: done in 88408c2
